Packs/Viper/Integrations/Viper/Viper.py

```diff
@@ -0,0 +1,181 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' IMPORTS '''
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class ViperClient(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def sample_information(self, file_hash):
+        '''Get Sample instance information from Viper'''
+        return self._http_request(
+            method='GET',
+            url_suffix=f'/malware/{file_hash}/'
+        )
+
+    def test_module(self):
+        return self._http_request(method='GET', url_suffix='/')
+
+    def sample_download(self, file_hash):
+        '''Download Sample from Viper'''
+        return self._http_request(
+            method='GET',
+            url_suffix=f'/malware/{file_hash}/download'
+        )
+
+
+def test_module(client):
+    """
+    Returning 'ok' indicates that the integration works like it suppose to. Connection to the service is successful.
+
+    Args:
+        client: Viper client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test
+    """
+    client.test_module()
+    return 'ok'
+
+
+def sample_download(file_hash):
+    api_key = demisto.params().get('apikey')
+    viper_project = demisto.params().get('viper_project')
+    base_url = urljoin(demisto.params()['url'], f'/api/v3/project/{viper_project}')
+    verify_certificate = not demisto.params().get('insecure', False)
+    url = f'{base_url}/malware/{file_hash}/download/'
+    authorization = f'Token {api_key}'
+    try:
+        sample = requests.get(url, verify=verify_certificate, headers={
+                              'Authorization': authorization, 'Accept': 'application/json'})
+
+    except Exception as e:
+        # demisto.error(traceback.format_exc())
+        return_error(f'Failed to execute {demisto.command()} command.\nError:\n{str(e)}')
+
+    return sample
+
+
+def sample_search(file_hash):
```

Comment:
Seems that there is no use of this function, it might be better to remove it

Reply:
Yes, your right,
I deleted the function.